documentation/en/mining.md

```diff
@@ -99,3 +99,23 @@ This env var can be used with `lotus-storage-miner`, `lotus-seal-worker`, and `l
 ### `FIL_PROOFS_USE_GPU_COLUMN_BUILDER=1` Environment variable
 
 This env var can be used with `lotus-storage-miner`, `lotus-seal-worker`, and `lotus-bench` to enable experimental precommit2 GPU acceleration
+
+### Setting multiaddresses
+
+It it possible to set multiaddresses for the miner to listen on in a miner's `config.toml` file 
```

Comment:
I would be even more pushy about this step, ie 

"To improve ____ it is recommended to set multiaddresses..." or even (default everyone to this step) "Next, set multiaddresses...."

Reply:
Adjusted!